Zend/tests/closures/closure_get_current.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+Closure::getCurrent()
+--FILE--
+<?php
+
+$i = 1;
+
+$c = function ($p) use (&$i) {
+    $self = Closure::getCurrent();
```

Comment:
Magic __CLOSURE__ constant might be more readable.

Reply:
But it's not a constant. It's an object. Which is different for every instance of the closure.

Reply:
good point!

Reply:
magic constants don't need to be constant anyway, they're magic. 
But I prefer Closure::getCurrent() to `__CLOSURE__` anyway because we may want other Closure-stuff in the future, and it's easy to extend Closure::*